### ISSUE TITLE
مشکل در اندروید ۱۱ و ۱۲

### ISSUE THREAD

**Issue by Chobin-113** [cafebazaar/unitypurchasing#5]
پرداخت درون برنامه ای در اندرویدهای ۱۰ به پایین کامل اجرا میشود ولی برای اندروید ۱۱ و بالاتر با مشکل مواجه می شود از Target API 29 به بالا پستیبانی نمی شود

**Comment by afshinunity** [comment]
تونستید مشکلتون رو حل کنید؟ چطوری؟
@Chobin-113

**Comment by Chobin-113** [comment]
سلام دوست عزیز
اپلیکیشن شما روی اندروید ۱۱و ۱۲ نصب میشه ولی پرداخت درون برنامه ای اجرا
نمیشه؟
اگه مشکلتون این هست اول از همه sdk تون رو به بالاترین ورژن یعنی ۳۳ فکر کنم
الان باشه آپدیت کنید بعد داخل target api قرار بدید automaticly highest
بعد دوباره خروجی بگیرید من این کار ها رو انجام دادم بعد اپلیکیشن رو نصب
کردم بعد از دو یا سه روز خود به خود درست شد

**Comment by afshinunity** [comment]
ما یه بازی منتشر کردیم با یونیتی 2022.3 با تنظیمات پیش فرض و هیچ مشکلی نداشتیم توی بیلد گرفتنش و منتشرش کردیم هم روی مایکت؛ بازار و همچنین گوگل پلی. گزینه automatically highest فعاله. بله روی اندروید 11-12 و 13 نصب و اجرا میشه بدون مشکل. تنها مشکل من استفاده از این پلاگین هستش که ایمپورتش کردم و تست کردم. هر چی میزنم روی دکمه خرید هیچ اتفاقی نمیوفته.

**Comment by Chobin-113** [comment]
برای gradle گرفتن از چه فیلترشکنی استفاده میکنید؟بعد آیا تبلیغات تپسل روی
اپلیکشن شما هست ؟

**Comment by Mehrdadgame** [comment]
@afshinunity 
@Chobin-113 
سلام دوستان برای من هم این مشکل دارد 

Error Unity Purchasing failed to initialize. Reason: NoProductsAvailable
برای من این ارور میدهد 
شما تونستید درست کنید؟

**Comment by afshinunity** [comment]
> 

بله. من از codeless استفاده نکردم و از اونی که از طریق کد هستش استفاده کردم ارور نداد و جواب گرفتم

**Comment by Mehrdadgame** [comment]
دوستان مشکل از کافه بازار !
این باگ وقتی میاد که شما یا کافه بازار روی گوشی نداشته باشید یا اینکه داخل اپ کافه بازار لاگین نکرده باشید .
کاقه بازار باید برای این یک باگ یک لاگ بهتر را نشون بده

**Comment by mblogest** [comment]
Attention!

Keep in mind, that builds in these versions, do not support android 11+ devices.

توی توضیحات نوشته شده که اندروید 11 به بالا ساپورت نمیشه. ما روی 13 هم مشکل نداریم. فقط 11 و 12 این مشکل رو دارن
